client/state/themes/selectors/has-atomic-transfer-dialog-accepted.js

```diff
@@ -0,0 +1,15 @@
+import 'calypso/state/themes/init';
+
+/**
+ * Checks if the atomic transfer dialog has been accepted.
+ *
+ * @param {Object} state   Global state tree
+ * @param {string} themeId Theme ID to check if the atomic transfer dialog has been accepted.
+ * @returns True if the atomic transfer dialog has been accepted. Otherwise, False.
+ */
+export function hasAtomicTransferDialogAccepted( state, themeId ) {
```

Comment:
Main comment: might it be clearer if we call this `wasAtomicTransferDialogAccepted()`? It still has the boolean prefix, but more clearly represents the check for a thing that already happened.

Secondary cleanup comments: Can we also add the `AppState` type here? And maybe enforce/explicitly note that the return is a `boolean`?